app/src/lib/file-system.ts

```diff
@@ -0,0 +1,20 @@
+import * as fileSystem from 'fs-extra'
+
+
+/** this function wraps the filesystem mkdir function and silently returns if
```

Comment:
Not sure if these docs quite reflect the current behaviour.

A rough draft, feel free to adapt from this:

```
Create a directory using the default `fs.mkdir` but ignore the failure that
occurs when the directory exists. All other errors need to be handled by the caller.
```

Don't forget the capital letters and full stops at the end of the sentence.